nucypher/policy/conditions/types.py

```diff
@@ -1,6 +1,66 @@
-from typing import Dict, List, Union
+import sys
 
-ConditionValues = Union[str, int, bool]
-ReturnValueDict = Dict[str, ConditionValues]
-ConditionDict = Dict[str, Union[ConditionValues, ReturnValueDict]]
-LingoList = List[ConditionDict]
+if sys.version_info >= (3, 8):
+    from typing import TypedDict
+else:
+    from typing_extensions import TypedDict
+
+from typing import Any, Dict, List, Type, Union
+
+from web3.types import ABIFunction
+
+#########
+# Context
+#########
+ContextDict = Dict[str, Any]
+
+
+################
+# ConditionLingo
+################
+
+#
+# OperatorDict represents:
+# - {"operator": "and" | "or"}
+class OperatorDict(TypedDict):
+    operator: str
```

Comment:
A stricter type that encompasses the above comment:
```suggestion 
    operator: Literal['and', 'or']
```